articles/connections/apple-siwa/configure-email-relay-service.md

```diff
@@ -0,0 +1,32 @@
+---
+title: Configure email relay service for Sign In with Apple
+description: Learn how to configure Apple's email relay service functionality for your app. 
+toc: true
+topics:
+  - authentication
+  - connections
+  - social
+  - apple
+  - email
+contentType: how-to
+useCase:
+  - add-login
+  - customize-connections
+  - configure-email-relay-service
+---
+## Configure Email Relay Service for Sign In with Apple
+
+1. To use the email relay service that Apple provides, click **More** under the **Certificates, Identifiers, & Profiles** section and click **Configure**.
+2. Add your domain (`<YOUR CUSTOM DOMAIN>.com`) in the field available on **Domains and Associated Email Addresses**.
```

Comment:
Mentioning Custom Domain here can be confusing because they might assume if the auth0 custom domain. I would rather be explicit saying:
```suggestion
2. Add the domain of the email addresses you will use to contact users (`<YOUR EMAIL DOMAIN>.com`) in the field available on **Domains and Associated Email Addresses**.
```